SkateShop.Maui/MauiProgram.cs

```diff
@@ -1,36 +1,37 @@
-﻿using Microsoft.Extensions.DependencyInjection;
+﻿using CommunityToolkit.Mvvm.Messaging;
 using SkateShop.Maui.Entities.Cart;
 using SkateShop.Maui.Views;
 
 namespace SkateShop.Maui;
 
 public static class MauiProgram
 {
-	public static MauiApp CreateMauiApp()
-	{
-		var builder = MauiApp.CreateBuilder();
-		builder
-			.UseMauiApp<App>()
-			.ConfigureFonts(fonts =>
-			{
-				fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
-				fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
-			});
+    public static MauiApp CreateMauiApp()
+    {
+        var builder = MauiApp.CreateBuilder();
+        builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+            });
 
-		builder.Services.AddSingleton<CartStore>();
+        builder.Services.AddSingleton<IMessenger>(StrongReferenceMessenger.Default);
```

Comment:
Injecting this feels so much better, nice!

Although my only fear with injecting the `StrongReferenceMessenger` is if clients assume they have a `WeakReferenceMessenger` and then don't unsubscribe. I probably won't use `WeakReferenceMessenger` though, and I usually unsubscribe anyways. I still think injecting this makes sense though.